docs/extend/extend-apiml/api-medation-sso-integration-extenders.md

```diff
@@ -11,15 +11,17 @@ This article does not cover the client methods to call API ML and authenticate.
 
 To integrate with API Mediation Layer and leverage Single Sign On, choose from the following three possible methods:
 
-- Accepting JWT token (recommended)
-- Accepting SAF IDT token
-- Accepting PassTicket
+- [Accepting JWT token (recommended)](#accepting-jwt)
+- [Accepting SAF IDT token](#accepting-saf-idt)
+- [Accepting PassTicket](#accepting-passtickets)
 
-Two additional possibilities can potentially be leveraged to enable Single Sign On but are not properly integrated with the standard API ML:
+Additional possibilities can potentially be leveraged to enable Single Sign On but are **not** properly integrated with the standard API ML:
 
-- Bypassing the authentication for the service
-- Asking for details about the x509 certificate used for authentication  
-**Note:** Asking for details about the x509 certificate does not properly participate in SSO as this method cannot accept all authentication methods that are supported upstream of API Mediation Layer. 
+* [Bypassing authentication for the service](#bypassing-authentication-for-the-service)   
+    **Note:** This option is for SSO only if the service does not have an authenticated endpoint.
```

Comment:
"SSO" is introduced here but above we don't see it defined as "single sign-on (SSO)"